db/migrate/20191016143953_fruits.rb

```diff
@@ -1,11 +0,0 @@
-class Fruits < ActiveRecord::Migration[5.2]
```

Comment:
Are you sure that you want to delete this migration file?

Reply:
it was a duplicate so yes